exercises/yacht/src/Yacht.hs

```diff
@@ -0,0 +1,17 @@
+module Yacht (yacht, Category(..)) where
+
+data Category = Ones
+              | Twos
+              | Threes
+              | Fours
+              | Fives
+              | Sixes
+              | FullHouse
+              | FourOfAKind
+              | LittleStraight
+              | BigStraight
+              | Choice
+              | Yacht
+
+yacht :: Category -> [Int] -> Int
+yacht _ _ = error "You need to implement this function."
```

Comment:
according to `grep _ exercises/*/src/*.hs` we have never used `_` in a stub. I am guessing we should provide some name instead of `_` here, unless there is a reason to deviate from the existing wisdom. However if the existing wisdom and a reason for it was documented, I seem to have lost the link to the documentation.

Reply:
Very good point.  I didn't base that decision off of looking at the other work.

I just followed the convention of not naming variables that wouldn't be used.

I see now that we are not suppose to suggest how many variable bindings they use.
That makes plenty of sense in Haskell.  I guess I expected it not to compile if nothing was accepted.  Thank you for pointing this out :smile:

Reply:
Pushed new changes which make the stub conform with other exercise stubs ;)

Reply:
Our current convention is to just name the input variables meaningfully. This does provide linting warnings. One could prevent those linting errors by prefixing those meaningful names with `_`s, but this does not occur in any exercise stubs, either, and would probably cause a little confusion.